src/main.js

```diff
@@ -2,7 +2,15 @@ function convertItemsToUnorderedList(listItems) {
   const ulElement = document.createElement("ul");
   listItems.forEach(item => {
     const liElement = document.createElement("li");
-    liElement.textContent = item.name;
+    liElement.classList.add("fs-api-entry");
```

Comment:
You can squash these two .add calls into one 😄: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods